lib/mix/lib/mix/tasks/deps.tree.ex

```diff
@@ -20,6 +20,10 @@ defmodule Mix.Tasks.Deps.Tree do
     * `--exclude` - exclude dependencies which you do not want to see printed. You can
       pass this flag multiple times to exclude multiple dependencies.
 
+    * `--include` - only include dependencies which you want to see printed. You can
```

Comment:
It should probably be called `--only`, to mirror `mix test`.

Reply:
Actually, `--only` is not a good name, because you want its children. 🤔 Maybe there is another API we can mirror though.

Reply:
`mix xref` calls it `source`, which means only use those as root.

Reply:
`--only` already filters deps on a specific Mix env (to mirror `only: ...` in the deps list). I picked `--include` because it's the opposite of the already-existing `--exclude` flag.